omise/omise.php

```diff
@@ -0,0 +1,44 @@
+<?php
+if (! defined('_PS_VERSION_'))
+    exit();
+
+class Omise extends PaymentModule
+{
+
+    /**
+     * The name that will be display to the user at the back-end.
+     *
+     * @var string
+     */
+    const MODULE_DISPLAY_NAME = 'Omise';
+
+    /**
+     * The name that used to referece in the program.
+     *
+     * @var string
+     */
+    const MODULE_NAME = 'omise';
+
+    /**
+     * The version of the module.
+     *
+     * @var string
+     */
+    const MODULE_VERSION = '1.6.0.0';
+
+    public function __construct()
+    {
+        $this->name = self::MODULE_NAME;
```

Comment:
Readability is subjective. For me both styles have readability.

That video is nice but I did not use Sublime3 and how about other developers use other text editor or IDE.

I care about other developers that need to consistent this extended style.

This point is very small but if it matter to you, you can change it.